src/renderer.rs

```diff
@@ -622,13 +620,10 @@ fn page_header(title: &str, file_upload: bool, favicon_route: &str, css_route: &
 /// Converts a SystemTime object to a strings tuple (date, time)
 /// Date is formatted as %e %b, e.g. Jul 12
 /// Time is formatted as %R, e.g. 22:34
-fn convert_to_utc(src_time: Option<SystemTime>) -> Option<(String, String)> {
-    src_time.map(DateTime::<Utc>::from).map(|date_time| {
-        (
-            date_time.format("%b %e").to_string(),
-            date_time.format("%R").to_string(),
-        )
-    })
+fn convert_to_utc(src_time: Option<SystemTime>) -> Option<String> {
+    src_time
+        .map(DateTime::<Local>::from)
+        .map(|date_time| date_time.format("%Y/%m/%d %H:%M:%S %:z").to_string())
```

Comment:
Let's have these be displayed in the more traditional dash separated format:
```suggestion
        .map(|date_time| date_time.format("%Y-%m-%d %H:%M:%S %:z").to_string())
```
Alternatively, you could also use `to_rfc2822()` which might be more readable to some people. Your call.